test/integration/VirtMemTests.cc

```diff
@@ -13,15 +13,16 @@ const std::string configStr =
     "Timer-Frequency: 100, Micro-Operations: True, "
     "Vector-Length: 512, Streaming-Vector-Length: 512},"
     "Process-Image: {Heap-Size: 100000, Stack-Size: 100000, Mmap-Size: "
-    "200000}, Simulation-Memory: {Size: 500000}, CPU-Info: "
+    "200000}, Memory-Hierarchy: {Cache-Line-Width: 64, L1-Data: "
+    "{Access-Latency: 1, Size: 500000}}, CPU-Info: "
     "{Generate-Special-Dir: False}}";
 
 TEST(VirtMemTest, MmapSysCallNoAddressNoFile) {
   Config::set(configStr.c_str());
 
   // Create simulation memory.
```

Comment:
Maybe expand the comment by saying that we use cache params for simulation memory just so that testing is faster

Reply:
Please expand in each test

Reply:
Changed config to use DRAM rather than L1 to avoid confusion